SEUNAGBEYE/More-Recipes
# #155899653 Design Email Templates

#### What does this PR do?
>- Design template for emails
#### Description of Task to be completed?
>- Design emails template
#### How should this be manually tested?
>- git clone this project `git clone https://github.com/SEUNAGBEYE/More-Recipes.git`
>- run `npm install` to install all dependencies
>- run `npm run dev-server` to start the development server
-> register and you should get a mail

#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- #155899653
#### Screenshots (if appropriate):
<img width="581" alt="screen shot 2018-03-12 at 8 44 45 pm" src="https://user-images.githubusercontent.com/25517388/37305881-4322f270-2636-11e8-889e-c76f496549e3.png">
#### Questions:
>- N/A
